src/syntax/syntax.rkt

```diff
@@ -204,7 +331,25 @@
 
 (define-operator (cast real) real [ival identity])
 
+;; Accelerators
+
+(define-operator (expm1 real) real [spec (lambda (x) (- (exp x) 1))])
```

Comment:
I guess this is fine, but long term we def don’t want these here right?

Reply:
Long-term, something like `expm1` is an operator implementation. Operators are only Rival-supported / atomic operations.